### ISSUE TITLE
4k or high res monitor support

### ISSUE THREAD

**Issue by clearwaterstream** [jfchapman/VUPlayer#13]
I've recently switched to a new monitor with a native res of 4096x2304 and the app looks tiny (fonts and buttons, menu font is fine however). Any plans to support high dpi displays?

Thanks!

![image](https://user-images.githubusercontent.com/25708965/101172237-ed15c100-360e-11eb-9292-1ca098d57b28.png)

![image](https://user-images.githubusercontent.com/25708965/101172292-facb4680-360e-11eb-8492-11b8e84ea777.png)

**Comment by jfchapman** [comment]
The application is DPI aware, however there is a problem with the way the font settings are stored in the database. It only saves a single setting, so when you change the Windows scaling, it will still be using the old font. You can of course change the font sizes in the application (there are separate settings for the tree, list and counter fonts), but obviously it's not great when switching the Windows scaling back and forth. I will need to fix the way the font settings are stored to resolve this issue.

For the toolbar, this is scaling as I would expect (if it wasn't, it would be really tiny!). However, I have had a request to support larger toolbar buttons in general, so this is another pending change in the pipeline.

**Comment by jfchapman** [comment]
For version 4.10, I've added a fix which scales the font sizes on application start up if the desktop scaling has changed since the last time the application was run. So, from version 4.10 onward, this should preserve the relative font sizes when changing desktop scaling (although it won't do a 'first-time' fix for any font sizes that are already stored in the application settings).
I've also added some small, medium & large toolbar options.
